### DISCUSSION TITLE
Student Status Rejection

### DISCUSSION THREAD

**Discussion by gawlas** [community/community#51317] [GitHub Education]
Hi,

My student applications are rejecting all the time, I'm trying to get verified from yesterday, I've tried about 20 times, I upload my student ID (front and back), payment recipe, syllabus, mobile ID and still rejected.

Can someone take a look or help me with verification process? 

Maybe the problem is that on polish student ID, "Valid Unitl" date is on the back side of ID on little stickers, and rest of personal informations are on front side.

**Comment by samus-aran** [comment]
Hi @Gawlas  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
